string baader_action
---
string baader_response
bool baader_error
